packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.css

```diff
@@ -1,12 +1,60 @@
-.text_content {
-    padding: 25px;
-    padding-top: 50px;
+.total_quantity {
     font-weight: bold;
 }
 
-.container {
-    min-height: 20rem;
+.show_all_items_button {
+    composes: root_normalPriority from '../../Button/button.css';
+    width: 100%;
+    border: none;
+}
+
+.show_all_items_button > span {
```

Comment:
Avoid targeting tagnames—ever. In this case, it's not necessary anyway, because the element you're targeting will inherit `font-size` from its parent, the button.

```suggestion
.show_all_items_button {
```